src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvoker.java

```diff
@@ -172,22 +172,22 @@ private AboutToSubmitCallbackResponse validateAndSetFromAboutToSubmitCallback(fi
 
         validateSignificantItem(aboutToSubmitCallbackResponse, callbackResponse);
         callbackService.validateCallbackErrorsAndWarnings(callbackResponse, ignoreWarning);
+        //it will work out the state value based on state priority.
```

Comment:
we should avoid commenting code and replace comments with self explanatory method names.